apps/modernization-api/src/test/resources/features/patient/PatientDelete.feature

```diff
@@ -1,22 +1,77 @@
 @patient-delete
-Feature: I can send a patient delete request
-
-  Scenario: I can send a patient delete request
-    Given I am logged into NBS
-    And I have the authorities: "DELETE-PATIENT,VIEW-PATIENT" for the jurisdiction: "ALL" and program area: "STD"
-    And I have a patient
-    When I send a patient delete request
-    Then the delete request is posted to kafka
-
-  Scenario Outline: I cant send delete requests without the proper permissions
-    Given I am logged into NBS
-    And I have the authorities: "<authorities>" for the jurisdiction: "ALL" and program area: "STD"
-    And I have a patient
-    When I send a patient delete request
-    Then I get an access denied exception for patient delete
+Feature: Patient Delete
+
+  Background:
+    Given I have a patient
+    And I am logged into NBS
+
+  Scenario: I can delete a patient that is not associated with events
+    Given I have the authorities: "DELETE-PATIENT,VIEW-PATIENT" for the jurisdiction: "ALL" and program area: "STD"
+    When I delete the patient
+    Then the patient is deleted
+
+  Scenario Outline: I delete a patient without the proper permissions
```

Comment:
Think this should just be `Scenario` instead of `Scenario Outline`

Reply:
Either should work.  Intellij shows an error if you have a Scenario with examples but it still runs.  The test shows up in the report if set as a `Scenario` or a `Scenario Outline`